handlers/server_spec_handler.go

```diff
@@ -66,6 +68,15 @@ func CheckLatestVersion(serverVersion string) (*latest.CheckResponse, error) {
 	// If user is running an outdated release, let them know.
 	if res.Outdated {
 		logrus.Info("\n", serverVersion, " is not the latest Meshery release. Update to v", res.Current, ". Run `mesheryctl system update`")
+		logrus.Info("\nWould you like to upgrade to v", res.Current, " now? (y/n)?")
```

Comment:
@uds5501 https://github.com/manifoldco/promptui - A library that implements clean prompt. Issue is that not just y but we should accept and empty `enter` or `yes` as an answer.

Reply:
@kumarabd sounds good, will fix that!

Reply:
Good work, @uds5501. Let’s use only one question mark.